test/models/concerns/strippable_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class StrippableTest < ActiveSupport::TestCase
+  test 'should strip trailing whitespaces on input fields' do
+    class ExampleClass
+      include Strippable
+      attr_accessor :username, :email, :phone
```

Comment:
I like that you used a new class to test this module, rather than make use of `Book` :) Makes the test suite more reliable I think 👍